tests/playwright-test/config.spec.ts

```diff
@@ -304,7 +303,7 @@ test('should filter by project and allow passing RegExp start/end flags', async
       test('one', async ({}) => {
         console.log('%%' + test.info().project.name);
       });    `
-  }, { project: '/fooBar$/' });
+  }, { '--project-grep': '/fooBar$/' });
```

Comment:
I'd say with a separate option we can probably omit `//`.

Reply:
this matches our --grep behavior